src/docs/admin/tutorial/AdminTutorials.adoc

```diff
@@ -0,0 +1,40 @@
+= Tutorials: ProActive Workflows & Scheduling
+include::../../common-settings.adoc[]
+:toc-title: Tutorials for Administrators
+
+= Step by Step Tutorials
+
+== Run As Me
+This is a step by step tutorial, which was designed and tested for Ubuntu 14.04.
+
+ProActive Workflows & Scheduling allows advanced user management with the “run as me” feature. Run as me allows
+to execute a task under the current ProActive user, if an equally named system user exists on the executing machine.
+
+*Following steps configure the Node with a sample user called Gord:*
```

Comment:
I think using same username and group as ones created and used by agent-linux would help users, especially if they use agents.